delta/kernel/src/main/scala/ch/epfl/bluebrain/nexus/delta/kernel/kamon/KamonMonitoringCats.scala

```diff
@@ -62,14 +62,15 @@ object KamonMonitoringCats {
       component: String,
       tags: Map[String, Any] = Map.empty,
       takeSamplingDecision: Boolean = true
-  )(io: IO[A]): IO[A] =
+  )(io: IO[A]): IO[A] = {
     if (enabled)
       buildSpan(name, component, tags).bracketCase(_ => io) {
         case (span, ExitCase.Completed)    => finishSpan(span, takeSamplingDecision)
         case (span, ExitCase.Error(cause)) => failSpan(span, cause, takeSamplingDecision)
```

Comment:
The log may be better here ?

Reply:
I thought it would still make sense to log the errors even if Kamon is disabled 🤔